app/dfda/predictor-search/components/PredictorSearchResults.tsx

```
@@ -0,0 +1,240 @@
+          <div
+            key={study.id}
+            className="list card correlationResultList rounded-lg bg-white shadow-sm"
+            style={{ textAlign: "center" }}
+          >
+            <div className="item-body p-6">
+              {study.studyHtml?.studyHeaderHtml && (
+                <div
+                  onClick={() => {
+                    /* Add study page navigation */
+                  }}
+                  dangerouslySetInnerHTML={createMarkup(
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)